content/source/docs/cloud/overview.html.md

```diff
@@ -0,0 +1,147 @@
+
+### Remote Terraform Execution
+
+Terraform Cloud runs Terraform on disposable virtual machines in its own cloud infrastructure. (Remote Terraform execution is sometimes referred to as "remote operations.")
+
+Remote execution helps provide consistency and visibility for critical provisioning operations. It also enables powerful features like Sentinel policy enforcement, cost estimation, notifications, version control integration, and more.
+
+- More info: [Terraform Runs and Remote Operations](./run/index.html)
+
+#### Support for Local Execution
+
+[execution_mode]: ./workspaces/settings.html#execution-mode
+
+Remote execution can be disabled on specific workspaces with the ["Execution Mode" setting][execution_mode]. The workspace will still host remote state, and Terraform CLI can use that state for local runs via the [remote backend][].
+
+### Workspaces for Organizing Infrastructure
+
+Terraform's local workflow manages a collection of infrastructure with a _persistent working directory,_ which contains configuration, state data, and variables. Practitioners can use separate directories to organize infrastructure resources into meaningful groups, and Terraform will use content from whichever directory it is invoked from.
+
+Terraform Cloud organizes infrastructure with _workspaces_ instead of directories. Each workspace contains everything necessary to manage a given collection of infrastructure, and Terraform uses that content whenever it executes in the context of that workspace.
+
+- More info: [Workspaces](./workspaces/index.html)
+
+### Version Control Integration
+
+Infrastructure as code belongs in version control, so Terraform Cloud is designed to work directly with your version control system (VCS) provider.
+
+Each workspace can be linked to a VCS repository that contains its Terraform configuration, optionally specifying a branch and subdirectory. Terraform Cloud automatically retrieves configuration content from the repository, and will also watch the repository for changes:
+
+- When new commits are merged, affected workspaces automatically run Terraform plans with the new code.
+- When pull requests are opened, affected workspaces run speculative plans with the proposed code changes and post the results as a pull request check; reviewers can see at a glance whether the plan was successful, and can click through to view the proposed changes in detail.
+
+VCS integration is powerful, but optional; if you use an unsupported VCS or want to preserve an existing validation and deployment pipeline, you can use the API or Terraform CLI to upload new configuration versions. You'll still get the benefits of remote execution and Terraform Cloud's other features.
+
+- More info: [VCS-driven Runs](./run/ui.html)
+- More info: [Supported VCS Providers](./vcs/index.html#supported-vcs-providers)
+
+### Command Line Integration
```

Comment:
That's more of an implementation detail, part of the "how" rather than the "what." I'm fine with having people learn about that after they click through to find out more.